en/_posts/2016-06-21-deploying-wordpress-on-marathon.md

```diff
@@ -0,0 +1,87 @@
+
+![whitelist](/kb/images/2016-06-21-deploying-wordpress-on-marathon/whitelist.png)
+
+## Deploy the PhpMyAdmin manager
+
+We'll use the [https://hub.docker.com/r/phpmyadmin/phpmyadmin/](phpmyadmin/phpmyadmin) image, hosted on the Docker Hub.
+
+You'll need to configure the PhpMyAdmin so that it exposes the TCP container port 80 to the load balancer.
+
+![pma1](/kb/images/2016-06-21-deploying-wordpress-on-marathon/phpmyadmin.png)
+
+You then need to set environment variables, to define the host and port of your MySQL database:
+
+* `PMA_HOSTS`: the value listed in the [sunrise manager](https://www.ovh.com/manager/sunrise), section (Paas Database), in the project instance details page, under the name "Access URL"
+* `PMA_PORT`: the value listed in the [sunrise manager](https://www.ovh.com/manager/sunrise), section (Paas Database), in the project instance details page, under the name "Service port"
+
+Then, add some labels to [redirect your logs to the PaaS logs](/kb/en/docker/marathon-container-logs.html), and [configure the http vhost](/kb/en/docker/marathon-load-balancer.html) of your app.
+
+Finally, configure an [HTTP health-check](/kb/en/docker/marathon-health-checks.html) on the path `/`.
+
+![pma2](/kb/images/2016-06-21-deploying-wordpress-on-marathon/phpmyadmin2.png)
+
+Once your app is displayed as healthy in your marathon web interface, you should be able to access the PhpMyAdmin welcome page by requesting the configured vhost. In the previous screenshot, we set it to `pma.<username>.lb.<cluster>.containers.ovh.net`.
+
+![pma3](/kb/images/2016-06-21-deploying-wordpress-on-marathon/phpmyadmin-welcome.png)
+
+## Deploy the wordpress container
+
+We'll use the official [wordpress image](https://hub.docker.com/_/wordpress/), hosted on the Docker Hub.
+
+Similarly to PhpMyAdmin, we need to:
+
+* expose the TCP container port 80 on the LB
+* configure the container environment to define the DB host, port and credentials
+* add the [Paas Logs label](/kb/en/docker/marathon-container-logs.html) to be able to see the container logs in the PaaS Logs Graylog
+* configure the container vhost
+* configure the container HTTP health checks
+
+![wp1](/kb/images/2016-06-21-deploying-wordpress-on-marathon/wordpress.png)
+![wp2](/kb/images/2016-06-21-deploying-wordpress-on-marathon/wordpress2.png)
+
+Once again, you should be able to reach the wordpress welcome page by requesting the configured vhost.
+
+![wp-welcome](/kb/images/2016-06-21-deploying-wordpress-on-marathon/wordpress-welcome.png)
+
+# Conclusion
+
+You now have 3 stateless wordpress containers, communicating with a MySQL database, which you can manage from a web interface.
```

Comment:
Ah, good catch! I'll add this side note, and will remove it when we support stateful containers.